internal/cli/atlas/deployments/search/indexes/create.go

```diff
@@ -87,10 +112,43 @@ func (opts *CreateOpts) Run(ctx context.Context) error {
 	}
 
 	opts.index, err = db.CreateSearchIndex(ctx, opts.index.CollectionName, opts.index)
+	return err
+}
+
+func (opts *CreateOpts) RunAtlas() error {
+	if err := opts.validateAndPrompt(); err != nil {
+		return err
+	}
+
+	index, err := opts.NewSearchIndex()
 	if err != nil {
 		return err
 	}
-	return nil
+
+	opts.index, err = opts.store.CreateSearchIndexes(opts.ConfigProjectID(), opts.DeploymentName, index)
+	return err
+}
+
+func (opts *CreateOpts) Run(ctx context.Context) error {
+	if opts.DeploymentType == "" {
+		if err := opts.PromptDeploymentType(); err != nil {
+			return err
+		}
+	}
```

Comment:
we should ask for deployment type first

Reply:
sorry, got confused here, isn't this the first thing the Run method is doing?

Reply:
ops, you are right 🤦 resolving xD